tests/FSharp.Compiler.ComponentTests/FSharpChecker/FindReferences.fs

```diff
@@ -137,15 +137,17 @@ let ``We find back-ticked identifiers`` () =
             ])
         }
 
-[<Fact>]
-let ``We find operators`` () =
+[<Theory>]
+[<InlineData("++")>]
+[<InlineData(">=>")>]
+let ``We find operators`` operator =
     SyntheticProject.Create(
-        { sourceFile "First" [] with ExtraSource = "let (++) x y = x - y" },
-        { sourceFile "Second" [] with ExtraSource = """
+        { sourceFile "First" [] with ExtraSource = $"let ({operator}) x y = x - y" },
+        { sourceFile "Second" [] with ExtraSource = $"""
 open ModuleFirst
-let foo x = x ++ 4""" })
+let foo x = x {operator} 4""" })
         .Workflow {
-            placeCursor "Second" 8 16 "let foo x = x ++ 4" ["++"]
+            placeCursor "Second" 8 16 $"let foo x = x {operator} 4" [operator]
```

Comment:
@0101 
If I understand correctly the second number argument for placeCursor function should be at the end of the symbol. If that is true then I think the test is falling because the difference of length of the "++" operator and ">=>". If you increase 16 to 17 and adjust the expected values the test will pass for ">=>" .